dali/pipeline/operator/builtin/external_source.h

```diff
@@ -621,19 +614,9 @@ class ExternalSource : public Operator<Backend>, virtual public BatchSizeProvide
   WorkerThread sync_worker_;
 
  private:
-  using storage_t =
-      std::conditional_t<std::is_same<Backend, GPUBackend>::value,
-                         detail::CachingList<uptr_tl_type>, detail::CachingList<uptr_tv_type>>;
-
-  template <typename Be = Backend>
-  std::enable_if_t<std::is_same<Be, GPUBackend>::value, storage_t &> GetStorage() {
+  detail::CachingList<uptr_tl_type>& GetStorage() {
```

Comment:
Do we need this anymore? Can't we use `tl_data_` directly?

Reply:
done